ixmp4/db/utils/__init__.py

```diff
@@ -1,7 +1,8 @@
 from contextlib import suppress
-from sqlalchemy import sql, inspect
-from sqlalchemy.sql import ColumnCollection
+
+from sqlalchemy import inspect, sql
 from sqlalchemy.orm import Mapper
+from sqlalchemy.sql import ColumnCollection
```

Comment:
Importing both `sql` and `sql.ColumnCollection` explicitly seems a little redundant.